torch/csrc/autograd/init.cpp

```diff
@@ -49,8 +49,8 @@ PyObject* THPAutograd_initExtension(PyObject* _unused, PyObject *unused) {
       .def("has_cuda", &Event::has_cuda)
       .def("shapes", &Event::shapes);
 
-  m.def("_enable_profiler", enableProfiler);
-  m.def("_disable_profiler", disableProfiler);
+  m.def("_enable_profiler", [](const ProfilerConfig& config) { enableProfiler(config); });
```

Comment:
just curious: why do you need these wrappers? didn't the previous code work?

Reply:
yes, it doesn't like an argument with default value, I know we can also expose that with pybind, but there's no need to use it on python level